config/v1/types_scheduling.go

```diff
@@ -93,6 +96,23 @@ var (
 	NoScoring SchedulerProfile = "NoScoring"
 )
 
+// ProfileCustomizations contains various parameters for modifying the default behavior of certain profiles
+type ProfileCustomizations struct {
```

Comment:
In the future, how many other fields do you expect to be present here? Is it possible any will be mutually exclusive from one another?

Reply:
> how many other fields do you expect to be present here?

Hard to say. I can think of another one for changing the bin-packing weight.

> Is it possible any will be mutually exclusive from one another?

Only a single profile is active at the same time. It can happen though it's going to be limited.

Reply:
Ok maybe I should rephrase then, I see that the profile is selectable above, can the dynamic resource allocation field be set with each and every profile option in that enum?